tritonsrc/bwd_split_kernel.py

```diff
@@ -102,6 +109,34 @@ def bwd_kernel_dk_dv(
         order=(1, 0)
     )
     off_zh = off_z * num_h + off_h * 1
+    if BIAS_TYPE == 0:
+        B_block_ptr = 0
+        DB_block_ptr = 0
+    elif BIAS_TYPE == 1:
+        B_block_ptr = tl.make_block_ptr(
+                base=B + off_h * stride_bh + off_z * stride_bz,
+                shape=(seqlen_q, seqlen_k),
+                strides=(stride_bm, stride_bn),
+                offsets=(0, start_m),
+                block_shape=(BLOCK_M, BLOCK_N),
+                order=(1, 0)
+                )
+        if (stride_dbz == 0 or stride_dbh == 0) or stride_dbm == 0:
+            store_db = False
+        else:
+            store_db = True
+        # Still have to make one even if no_db = False
+        # due to a limit of Triton: runtime branches must have identical data types.
+        DB_block_ptr = tl.make_block_ptr(
```

Comment:
I see a store but no load so in the end what is this storing?

Reply:
D is the prefix for differentials, and DB is the differential of bias tensor.